src/main/java/com/streamforge/data/dto/twitch/UserDataNativeDto.java

```diff
@@ -0,0 +1,117 @@
+package com.streamforge.data.dto.twitch;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+import java.util.Date;
+
+public class UserDataNativeDto {
+
+    private String logo;
+
+    @JsonAlias("display_name")
+    private String displayName;
+
+    @JsonAlias("updated_at")
+    private Date updatedAt;
+
+    @JsonAlias("_id")
```

Comment:
WTF, who is sending the response with fields like this???

Reply:
Twitch)